lib/miq_expression/tag.rb

```diff
@@ -19,12 +19,24 @@ def contains(value)
     model.arel_attribute(:id).in(ids)
   end
 
+  def numeric?
+    false
+  end
+
   def column_type
     :string
   end
 
-  def equ?(other)
+  def sub_type
+    column_type
+  end
+
+  def attribute_supported_by_sql?
+    false
+  end
+
+  def eql?(other)
```

Comment:
tag_spec.rb uses equality for tests.

I defined `eql?` and aliased `==` to it. I'm cargo-culting equality from other ruby definitions that I've seen.